### PR TITLE
Amy supershort writeup for Ember Times 56

### PR DESCRIPTION
**Pull request by amyrlam**
First stab - may try to flesh out tmrw. Any feedback let me know!

I found it tough to say much about the podcast but could summarize the RFCs if needed. I also wasn't sure if it is indeed "splatributes"?

There is some double bracket curly syntax in here that may need to be removed also... will comment inline too.

### CODE REVIEW

File: source/blog/2018-07-20-the-ember-times-issue-56.md [hunk 1]
```diff
@@ -38,11 +38,11 @@ Would you like to use dynamic import() to lazy-load deps into your Ember app rig
 
 ## [SECTION TITLE](#section-url)
 
-
 ---
 
-## [SECTION TITLE](#section-url)
+## [RFC Roundup with @rwjblue](https://emberweekend.com/episodes/rfc-roundup-with-rwjblue)
 
+The Ember Weekend podcast is back on the mic! Want to hear more about some recent RFCs? @rwjblue discusses [RFC #276: Named Arguments Syntax (`{{@foo}}`)](https://github.com/emberjs/rfcs/blob/master/text/0276-named-args.md), [RFC #213: Custom Components API](https://github.com/emberjs/rfcs/pull/213), and [RFC #311: Introduce `<AngleBracketInvocationSyntax />`](https://github.com/emberjs/rfcs/blob/master/text/0311-angle-bracket-invocation.md) in [Ember Weekend's latest podcast](https://emberweekend.com/episodes/rfc-roundup-with-rwjblue). Entertaining and informative! TIL "splatributes". Subscribe to Ember Weekend [here](https://itunes.apple.com/us/podcast/ember-weekend/id981719021) on iTunes. 🎙
```
Comment (amyrlam) [inline source/blog/2018-07-20-the-ember-times-issue-56.md]: Would it be better to not write `{{@foo}}` at all with Goodbits?

Comment (jessica-jordan) [reply source/blog/2018-07-20-the-ember-times-issue-56.md]: Yes, good point, I think it makes it easier to not forget about removing it in goodbits and I think in this case you could also be able to convey the message by writing `@foo` on its own. What do you or @tylerturdenpants think?

Comment (tylerturdenpants) [reply source/blog/2018-07-20-the-ember-times-issue-56.md]: I prefer clarity over assumption. As for goodbits, I think we may need to add it the template to remember to fix it before submitting it.

File: source/blog/2018-07-20-the-ember-times-issue-56.md [hunk 2]
```diff
@@ -66,15 +66,8 @@ Would you like to use dynamic import() to lazy-load deps into your Ember app rig
...
-## [The Ember Times is What We Make It 🙌](https://embercommunity.slack.com/messages/C8P6UPWNN/)
-
-The **Ember Times** is a **weekly news editorial** featuring all the new things that are going on in Emberland.
-[Subscribe to our e-mail newsletter](https://the-emberjs-times.ongoodbits.com/) to get the next edition **right to your inbox**.
-And if you've always wanted to be an OSS journalist yourself,
-drop by [#topic-embertimes](https://embercommunity.slack.com/messages/C8P6UPWNN/)
-on the Ember Community [Slack Chat](https://ember-community-slackin.herokuapp.com/)
-and **write** the next edition of the Ember Times **together with us**!
-
+Have a suggestion for next week's Ember Times? Want to write for us? Pop into [#topic-embertimes](https://embercommunity.slack.com/messages/C8P6UPWNN/)
```
Comment (tylerturdenpants) [inline source/blog/2018-07-20-the-ember-times-issue-56.md]: @amyrlam Thanks for this update

**Pull request merged**